grunt/tasks/tracking-remove.js

```diff
@@ -1,34 +1,8 @@
 module.exports = function(grunt) {
-  var Helpers = require('../helpers')(grunt);
-  var path = require('path');
-
+  const Helpers = require('../helpers')(grunt);
   grunt.registerTask('tracking-remove', 'Removes all tracking IDs', function() {
     if (!Helpers.isPluginInstalled('adapt-contrib-spoor')) return;
-
-    var options = this.options();
-
-    function removeTrackingIds(blocksPath, coursePath) {
-      var blocks = grunt.file.readJSON(blocksPath);
-      var course = grunt.file.readJSON(coursePath);
-
-      for (var i = 0; i < blocks.length; i++) {
-        delete blocks[i]._trackingId;
-      }
-      delete course._latestTrackingId;
-      grunt.log.writeln('Tracking IDs removed.');
-      grunt.file.write(coursePath, JSON.stringify(course, null, 4));
-      grunt.file.write(blocksPath, JSON.stringify(blocks, null, 4));
-    }
-
-    var isOutputDir = (grunt.option('outputdir') && grunt.option('outputdir').slice(-5) !== 'build');
-    var sourcedir = isOutputDir ? grunt.option('outputdir') : grunt.config('sourcedir');
-
-    var blocksFiles = grunt.file.expand(path.join(sourcedir, options.blocksFile));
-    var courseFiles = grunt.file.expand(path.join(sourcedir, options.courseFile));
-
-    for (var i = 0; i < blocksFiles.length; i++) {
-      removeTrackingIds(blocksFiles[i], courseFiles[i]);
-    }
-
+    const data = Helpers.getFramework().getData();
```

Comment:
I know it's not part of your changes but I think the check for spoor in L4 should be removed.

Reply:
Will have a crack at this when I layout moving tracking ids from blocks to components.